kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/status/NetworkStatusServiceAdapter.java

```diff
@@ -337,6 +337,12 @@ private void setWifiStateProperties(GwtNetInterfaceConfig gwtNetInterfaceConfig,
             } else if (wifiInterfaceInfo.getMode() == WifiMode.INFRA) {
                 setWifiInfraStateProperties(gwtWifiNetInterfaceConfig, wifiInterfaceInfo);
             }
+
+            if (logger.isDebugEnabled()) {
+                Optional<WifiAccessPoint> activeAP = wifiInterfaceInfo.getActiveWifiAccessPoint();
+                activeAP.ifPresent(accessPoint -> logger.debug("Wifi Interface Info: {}",
```

Comment:
Maybe add a more meaningful message, like `WiFi current channel status info: {}`, or just remove the logging